styles.css

```diff
@@ -157,19 +166,29 @@ img {
 }
 
 .work h2 {
-  width: 327px;
+  max-width: 327px;
```

Comment:
[OPTIONAL] Kindly remove the max-width from here in the Desktop version so that the text can stay on one line 😀

![image](https://user-images.githubusercontent.com/73714615/186664355-605ac6f3-cf67-4b34-acc4-6f5dc690318a.png)